src/components/ha-assist-chat.ts

```diff
@@ -189,12 +201,23 @@ export class HaAssistChat extends LitElement {
     `;
   }
 
-  private _scrollMessagesBottom() {
-    const scrollContainer = this._scrollContainer;
-    if (!scrollContainer) {
-      return;
+  private async _scrollMessagesBottom() {
+    const lastChatMessage = this._lastChatMessage;
+    if (!lastChatMessage.hasUpdated) {
+      await lastChatMessage.updateComplete;
+    }
+    if (
+      this._lastChatMessageImage &&
+      !this._lastChatMessageImage.naturalHeight
+    ) {
+      await this._lastChatMessageImage.decode();
```

Comment:
`decode` can throw an exception. Wrap it in a `try`/`catch` and log the exception. Don't think we need more complex error handling